docs/config.md

```diff
@@ -598,6 +598,46 @@ paginate = "auto"
 paginate = "never"
 ```
 
+### Builtin pager
+
+Our builtin pager is based on
+[`streampager`](https://github.com/markbt/streampager/) but is configured within
+`jj`'s config. It is configured via the `ui.streampager` table.
+
+#### Wrapping
+
+Wrapping performed by the pager happens *in addition to* any
+wrapping that `jj` itself does.
+
+```toml
+[ui.streampager]
+wrapping = "anywhere"  # wrap at screen edge (default)
+wrapping = "word"      # wrap on word boundaries
+wrapping = "none"      # strip long lines, allow scrolling
+                       # left and right like `less -S`
+```
+
+#### Auto-exit, clearing the screen on startup or exit
+
+You can configure whether the pager clears the screen on startup or exit, and
+whether it quits automatically on short inputs. When the pager auto-quits,
+features like word-wrapping are disabled.
+
+
+```toml
+[ui.streampager]
+# Do not clear screen on exit. Use a full-screen interface for long
+# output only. Like `less -FX`.
+interface = "quit-if-one-page"  # (default).
+# Always use a full-screen interface, ask the terminal to clear the
+# screen on exit. Like `less +F +X`.
```

Comment:
Maybe you meant `less` or `less -+FX`? Appears that `+` takes command name.

Reply:
Thank you! TIL. I was wondering why `less -F +X` didn't do *quite* what I thought it should, but I convinced myself this was because of some weird precedence issue between my `LESS` and the arguments. 😆 

This should indeed be `less -+FX`. I'm not a huge fan of this syntax, but at least it works.